src/PlusDataCollection/Atracsys/vtkPlusAtracsysTracker.cxx

```diff
@@ -236,6 +232,81 @@ PlusStatus vtkPlusAtracsysTracker::InternalConnect()
   // get device type
   this->Internal->Tracker.GetDeviceType(this->Internal->DeviceType);
 
+  // helper to translate option names from Plus nomenclature to Atracsys' one
+  auto translateOptionName = [this](const std::string & optionName, std::string & translatedOptionName)
+  {
+    std::map<std::string, std::string>::const_iterator itt =
+      this->Internal->DeviceOptionTranslator.find(optionName);
+    if (itt == this->Internal->DeviceOptionTranslator.cend())
+    { return false; }
+    else
+    {
+      translatedOptionName = itt->second;
+      return true;
+    }
+  };
+
+  // handling options that are used only internally
+  std::map<std::string, std::string>::const_iterator itd;
+  // ------- time allocated for activer marker pairing
+  itd = this->Internal->DeviceOptions.find("ActiveMarkerPairingTimeSec");
+  if (itd != this->Internal->DeviceOptions.cend())
+  {
+    if (strToInt32(itd->second, this->Internal->ActiveMarkerPairingTimeSec))
+    {
+      if (this->Internal->ActiveMarkerPairingTimeSec > 15)
+      {
+        LOG_WARNING("Marker pairing time is set to " << this->Internal->ActiveMarkerPairingTimeSec << "seconds, tracking will not start until this period is over.");
```

Comment:
IMO this should be converted to a LOG_INFO and always shown regardless of pairing time duration

Reply:
Good point, I've changed this.